src/components/mobile/ListPage.js

```diff
@@ -0,0 +1,102 @@
+import { useRect } from '@reach/rect'
+import { useEffect, useRef, useState } from 'react'
+import styled from 'styled-components/macro'
+
+import { useMap } from '../../contexts/MapContext'
+import { getLocations } from '../../utils/api'
+import InfiniteList from '../list/InfiniteList'
+// TODO: Ask Jeffrey what the limit for mobile list view should be
+const LIMIT = 30
+
+const ListPageContainer = styled.div`
+  height: 100%;
+`
+
+const LoadingContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+`
+
+const ListPage = () => {
+  const { view } = useMap()
+  const [locations, setLocations] = useState([])
+  const [hasMoreItems, setHasMoreItems] = useState(false)
+  const [isNextPageLoading, setIsNextPageLoading] = useState(false)
+
+  const container = useRef()
+  const rect = useRect(container) ?? { width: 0, height: 0 }
+
+  const fetchListEntries = async (bounds, center, offset) => {
+    // TODO: Include filters by consolidating all getLocations API calls
+    const locations = await getLocations({
+      swlng: bounds.sw.lng,
+      nelng: bounds.ne.lng,
+      swlat: bounds.sw.lat,
+      nelat: bounds.ne.lat,
+      lng: center.lng,
+      lat: center.lat,
+      limit: LIMIT,
+      offset,
+    })
+    return locations
+  }
+
+  useEffect(() => {
+    const fetchInitialListEntries = async () => {
+      const { bounds, zoom, center } = view
+      if (bounds?.ne.lat != null && zoom > 12) {
```

Comment:
I think the `!=` (equality) here is pretty ambiguous. This evaluates to `bounds?.ne.lat && zoom > 12`, so use this instead. If you meant to only check if `bounds?.ne.lat` is not `null` then use an identity comparison (`!==`)

Reply:
Ah yeah Lauren might have copied the example from MapPage, which also uses !=. Siraj I remember you commented on this potentially being a problem if `lat === 0` 😝